pkg/plugins/app_plugin.go

```diff
@@ -15,7 +15,8 @@ import (
 
 type AppPlugin struct {
 	FrontendPluginBase
-	Routes []*AppPluginRoute `json:"routes"`
+	Routes   []*AppPluginRoute `json:"routes"`
+	NoConfig bool              `json: "noConfig"`
```

Comment:
Maybe `noConfigPage` or `noConfigPageInNav` instead? `noConfig` feels very generic.

Reply:
yea had the same thought